resources/accessibility/storefront/index.md

```diff
@@ -0,0 +1,114 @@
+---
+nav:
+  title: Storefront
+  position: 30
+---
+
+# Accessibility in the Storefront
+
+At Shopware, we are committed to creating inclusive and barrier-free shopping experiences for our merchants and their customers.
+
+## What shopware does to ensure accessibility
+
+* Shopware is committed to fulfill the [WCAG 2.1 AA](https://www.w3.org/TR/WCAG21/) accessibility guidelines and Barrier-Free Information Technology Regulation (BITV 2.0) in the Storefront.
+* The Storefront is using [Bootstrap components](https://getbootstrap.com/docs/5.3/getting-started/accessibility/) that already consider good accessibility practices, for example using aria roles.
+* Much of the HTML structure and CSS styling already fulfill accessibility guidelines. However, there are still [open accessibility issues](#Overview-of-known-accessibility-issues) that will be addressed. 
+* Automated [E2E testing with playwright](https://github.com/shopware/shopware/tree/trunk/tests/acceptance) and axe reporter are used to ensure future accessibility.
+
+## How core accessibility improvements are released
+
+Accessibility improvements are rolled out in regular minor releases, similar to other improvements or bug-fixes. We implement all accessibility improvements in the current major version `6.6.x` and also `6.5.x`.
+There is no large "accessibility release" planned that ships all accessibility improvements at once. 
+
+## How to deal with breaking accessibility changes
+
+Ensuring an accessible shop page can require changes in the HTML/Twig structure or the CSS. This can cause unintended behavior for an extension that is modifying an area that is being changed to improve accessibility.
+
+Because of this, breaking accessibility changes are not enabled by default. All accessibility changes that include breaking changes are implemented behind a feature flag: 
+
+```env
+ACCESSIBILITY_TWEAKS
+```
+
+However, breaking accessibility changes are still released regularly inside minor releases. They are just not active by default to not cause a breaking change.
+
+The feature flag `ACCESSIBILITY_TWEAKS` can be activated inside your `.env`, similar to the major feature flags like `V6_7_0_0`. When the feature flag is enabled, all available accessibility improvements are activated.
+This allows you to check if your project or extension is effected by the change and already prepare an adaptation to the change if it is necessary.
+
+::: warning
```

Comment:
I'm not sure how this is supposed to look, but it looks a bit strange in the markdown preview

Reply:
Yeah this is looking weird here. 😆 In GitHub there is no preview for this special syntax. In the real docs it will look like this:

![image](https://github.com/shopware/docs/assets/8480203/d71efdf2-6ff5-45fb-a93f-9e3d7a16be95)

Reply:
We will add a guide soon to the docs README how to preview dev docs locally 

Here is the current one:

https://github.com/shopware/developer-portal/blob/main/QUICK-START.md